experimenter/tests/integration/nimbus/test_remote_settings.py

```diff
@@ -257,30 +256,32 @@ def test_rollout_live_update_reject_on_experimenter(
     summary_page.wait_for_rejection_notice_visible()
 
 
-@pytest.mark.remote_settings
-@pytest.mark.skipif(
-    any(
-        app in os.getenv("PYTEST_ARGS")
-        for app in ["FOCUS_IOS", "IOS", "FENIX", "FOCUS_ANDROID", "DEMO_APP"]
-    ),
-    reason="Only run for non-mobile applications and non desktop",
-)
-def test_summary_release_date_not_visible(
-    selenium,
-    kinto_client,
-    create_experiment,
-    experiment_url,
-):
-    summary = create_experiment(selenium)
-    summary.launch_and_approve()
-
-    kinto_client.approve()
-
-    summary = SummaryPage(selenium, experiment_url).open()
-    summary.wait_for_live_status()
-
-    summary.wait_for_timeline_visible()
-    timeline_release_date = summary.wait_until_timeline_release_date_not_found()
-    assert not timeline_release_date
-    audience_release_date = summary.wait_until_audience_section_release_date_not_found()
-    assert not audience_release_date
+# @pytest.mark.remote_settings
+# @pytest.mark.skipif(
+#     not set(os.getenv("PYTEST_ARGS")).intersection(
+#         {
+#             BaseExperimentApplications.FIREFOX_FENIX.value,
+#             BaseExperimentApplications.FIREFOX_IOS.value,
+#             BaseExperimentApplications.FOCUS_ANDROID.value,
+#             BaseExperimentApplications.FOCUS_IOS.value,
+#         }
+#     ),
+#     reason="Only run for mobile applications",
+# )
+# def test_summary_release_date_displayed_for_mobile(
+#     selenium,
+#     kinto_client,
+#     create_experiment,
+#     experiment_url,
+# ):
+#     summary = create_experiment(selenium)
+#     summary.launch_and_approve()
+
+#     kinto_client.approve()
+
+#     summary = SummaryPage(selenium, experiment_url).open()
+#     summary.wait_for_live_status()
+
+#     summary.wait_for_timeline_visible()
+#     summary.wait_until_timeline_release_date()
+#     summary.wait_until_audience_section_release_date()
```

Comment:
Something wrong with this test?

Reply:
Oh yeah because of the way it's reading from os.environ I have to change that part, it'll be fixed before I move this out of draft 👍